README.md

```diff
@@ -5,7 +5,7 @@
 <!-- `gh_repo` is extracted via a special environment variable in GitHub Actions -->
 <!-- README.md is generated from README.Rmd. Please edit that file -->
 
-# cleanepi: Clean and standardize epidemiological data <img src="man/figures/logo.svg" align="right" width="130"/>
+# {{ packagename }}: Clean and standardize epidemiological data <img src="man/figures/logo.svg" align="right" width="130"/>
```

Comment:
@Degoot-AM - the `md` file does not need to be updated. All changes goes into the `Rmd` file. 
The github variables like `packagename` are only used in the `Rmd`. This is evaluated and the real name will appear in the `md` file.